### PR TITLE
Fix help modal BO responsive

### LINKED ISSUE #23016: FO - Responsive / Helpbox is not fully responsive with some mobile device format (ipad)
#### Describe the bug

With Tablet format the helpbox is a bit small in width

#### Expected behavior

The helpbox should be fully responsive and lisible

#### Steps to Reproduce

1. Go to 'BO/order" on Tablet device
2. Click on 'the help button
3. See error => the helpbox doesn't looks good

**Screenshots**
image

#### Additional information

* PrestaShop version: 1.7.8 & 1.7.7
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Help modal is unreadable on mobile
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23016.
| How to test?      | Go on a migrated page, open mobile responsive mode, open Floating button, click on help and see if it's readable, not like in the issue
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23072)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

File: admin-dev/themes/new-theme/scss/components/layout/_right-sidebar.scss [hunk 1]
```diff
@@ -1,96 +1,90 @@
...
-  -webkit-transform: translate(0,0);
-  -ms-transform: translate(0,0);
-  -o-transform: translate(0,0);
-  -moz-transform: translate(0,0);
-  transform: translate(0,0);
+  transform: translate(0, 0);
+}
+
+
+@media (max-width: 992px) {
```
Comment (eternoendless) [inline admin-dev/themes/new-theme/scss/components/layout/_right-sidebar.scss]: I think we have predefined breakpoints somewhere. Why don't you use those?

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/components/layout/_right-sidebar.scss]: You're right, forgot I swapped this file into scss one

It's fixed

File: admin-dev/themes/new-theme/scss/components/layout/_right-sidebar.scss [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  @include transition(transform 300ms ease);
+}
+
+.sidebar.sidebar-right {
+  transform: translate(100%, 0);
+}
+
+.sidebar.sidebar-right.sidebar-open {
+  transform: translate(0, 0);
+}
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/components/layout/_right-sidebar.scss]:
```suggestion
.sidebar.sidebar-right {
  transform: translate(100%, 0);
  &.sidebar-open {
    transform: translate(0, 0);
  }
}
```

Comment (Progi1984) [reply admin-dev/themes/new-theme/scss/components/layout/_right-sidebar.scss]: And all `.sidebar.sidebar-right` under this one.

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks

**Review by Progi1984 — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Comment by SD1982** [comment]
Hi @NeOMakinG 
It looks good on mobile
image

but still look like in the issue on tablet format
image

**Comment by NeOMakinG** [comment]
@SD1982 I did it on tablet too now

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by SD1982** [comment]
Looks good on tablet (vertical mode) now !!
image
Thanks @NeOMakinG

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @SD1982

**Pull request merged**